### ISSUE TITLE
Rex-Thinker-Plain experiment result.

### ISSUE THREAD

**Issue by shengyuwoo** [IDEA-Research/Rex-Thinker#3]
How should the prompts be designed for the Rex-Thinker-Plain experiment in terms of input and output format? The results I reproduced differ significantly from those in your paper. Could you provide examples of prompt designs used in the Rex-Thinker-Plain experiment, as well as hyperparameters like the learning rate during training?

**Comment by Mountchicken** [comment]
Hi @shengyuwoo 
During training, we randomly sample prompts from the following list (replace [OBJ] with the referring expression):
```python
[
    "Detect [OBJ].",
    "Please detect [OBJ] in this image.",
    "Find [OBJ] in the image.",
    "Detect [OBJ] in the image.",
    "Locate [OBJ] in the image.",
    "Identify [OBJ] in the image",
    "Please locate [OBJ].",
]
```

For training, we use the following hyperparameters:
```bash
--per_device_train_batch_size 4 
--gradient_accumulation_steps 4 
--learning_rate 1e-6 
--optim adamw_torch 
--warmup_ratio 0.03 
--weight_decay 0.01 
--max_grad_norm 1 
--lr_scheduler_type "cosine" 
```

**Comment by shengyuwoo** [comment]
Is the use of full-parameter training or just unfreeze the LLM model layer training?

**Comment by Mountchicken** [comment]
Only LLM is trainable

**Comment by shengyuwoo** [comment]
Ooooh, thanks, and one last question. Are you guys using Swift or LLaMa-Factory framework to train the Rex-Thinker-Plain model?

**Comment by Mountchicken** [comment]
We use a customized training framework based on LLaVA

**Comment by shengyuwoo** [comment]
I have another question: Is the Rex-Thinker-Plain model fine-tuned directly from the original LLaVA model, or did you modify LLaVA's architecture, conduct pre-training from scratch, and then fine-tune it? If you performed LLaVA pre-training from the beginning, approximately how many bounding-box-annotated samples were required?

**Comment by Mountchicken** [comment]
We directly fine-tune on Qwen2.5-VL-78B-Instruct.

**Comment by shengyuwoo** [comment]
Qwen2.5-VL-72B-Instruct or InternVL-78B-Instruct or Qwen2.5-VL-7B-Instruct ？

**Comment by Mountchicken** [comment]
Qwen2.5-VL-7B-Instruct. Sry for the typo
